Source/ASNetworkImageNode.mm

```diff
@@ -335,6 +343,10 @@ - (void)displayWillStartAsynchronously:(BOOL)asynchronously
 {
   [super displayWillStartAsynchronously:asynchronously];
   
+  if (_delegateFlags.delegateWillStartDisplayAsynchronously) {
```

Comment:
We don't grab a lock here to access the `_delegateFlags` and the `_delegate` here

Reply:
will update